test/cachex/actions/stats_test.exs

```diff
@@ -88,9 +88,10 @@ defmodule Cachex.Actions.StatsTest do
         hit_rate: 0.0,
         misses: 1,
         miss_rate: 100.0,
-        operations: 1,
+        operations: 2,
         calls: %{
-          get: 1
+          get: 1,
+          warm: 1
```

Comment:
I don't think I like that this counts here, so you can add this option to the initial warm call:

```
const(:notify_false) ++ [respect_async_warmers: true]
``` 

This should stop it affecting the statistics on startup. The option set `const(:notify_false)` basically exists as a way to re-use public APIs without it looking confusing to the user if they ever investigate.

Happy side effect that this diff gets a lot smaller too this way.

Reply:
The changes made allowed me to revert these spec changes